packages/contracts/test/exchange/dispatcher.ts

```diff
@@ -258,37 +254,12 @@ describe('AssetProxyDispatcher', () => {
             const erc20Balances = await erc20Wrapper.getBalancesAsync();
             const amount = new BigNumber(10);
             return expect(
-                assetProxyDispatcher.transferFrom.sendTransactionAsync(
-                    encodedProxyMetadata,
-                    makerAddress,
-                    takerAddress,
-                    amount,
-                    { from: exchangeAddress },
-                ),
-            ).to.be.rejectedWith(constants.REVERT);
-        });
-
-        it('should throw on transfer if requesting address is not authorized', async () => {
```

Comment:
Why was this test removed?

Reply:
There is no longer such a thing as an address that is authorized to call `dispatchTransferFrom`, it is simply an internal function within the Exchange contract.